### ISSUE TITLE
Multi Tenancy Support

### ISSUE THREAD

**Issue by BechirAhmed** [UniSharp/laravel-filemanager#1029]
Hi, thanks for the great work, actually I don't have an issue. 
I just want to know what are the possibilities to use this package with a multi-tenancy system?
Spacially this package [Tenancy for Laravel](https://tenancyforlaravel.com/), which has multiple types of identifications, like by domain, subdomain, Path & Request data.
Is that possible? 
if yes, How? Any ideas can help!

Thanks

**Comment by mrtsglk** [comment]
Any solution?

**Comment by BechirAhmed** [comment]
It works for me, but only with s3 file system.
I guess the solution is just make sure your app can upload and retrieve files from tenant folder located in storage and not from storage/app/public. That's why with s3 works fine because it doesn't look to storage app path, just for tenant folder and then files.

**Comment by mrtsglk** [comment]
@BechirAhmed Yes it only sees the tenant folder. But there is a preview problem. The url is not working correctly. My current preview url : http://localhost/storage/photos/. I want to change url.

**Comment by jaffarhussain1011** [comment]
still if you are facing this issue, you can use this setting 
![image](https://github.com/UniSharp/laravel-filemanager/assets/3069478/7cfb334d-9305-49b1-80e6-5ae63ee5ee69)
![image](https://github.com/UniSharp/laravel-filemanager/assets/3069478/25afe0b2-bdee-4da2-a389-82e2c99dcf57)

https://unisharp.github.io/laravel-filemanager/config
